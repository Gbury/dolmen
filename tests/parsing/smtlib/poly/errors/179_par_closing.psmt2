(set-logic ALL)
(declare-fun foo (par (A) () Bool _
