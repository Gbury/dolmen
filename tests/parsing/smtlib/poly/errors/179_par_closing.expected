File "tests/parsing/smtlib/poly/errors/179_par_closing.psmt2", line 2, character 34-35:
Error while parsing a function declaration, read the reserved word '_',
      but expected two closing parenthesis.
